esp/services/ws_smc/ws_smcService.cpp

```diff
@@ -2344,3 +2345,175 @@ void CWsSMCEx::setActiveWUs(IEspContext &context, IEspActiveWorkunit& wu, IEspAc
     }
 }
 
+static const char *LockModeNames[] = { "READ", "WRITE", "HOLD", "SUB" };
```

Comment:
No. The ecm has the built-in function to convert string input to enum, but, not convert the enum to string.